src/types.ts

```diff
@@ -453,6 +459,12 @@ export interface IStorage {
   getScanningPolicy(): Promise<AddressScanPolicy>;
   getScanningPolicyData(): Promise<AddressScanPolicyData>;
   setScanningPolicyData(data: AddressScanPolicyData | null): Promise<void>;
+
+  // Nano Contract methods
+  isNanoContractRegistered(ncKey: string): Promise<boolean>;
+  getNanoContract(ncKey: string): Promise<INcData | null>;
+  registerNanoContract(ncKey: string, ncValue: INcData): Promise<void>;
```

Comment:
Should we change the parameter name to `ncId` instead of `ncKey` to keep consistency with the other methods?

Reply:
Fixed at https://github.com/HathorNetwork/hathor-wallet-lib/pull/614/commits/f7f228e35c65a12e4ab9eade924898eb8c63348b